website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
 | [sys/replication/performance/primary/secondary-token](/vault/api-docs/system/replication/replication-performance#generate-performance-secondary-token) | POST              | Generate a performance secondary activation token                                                                                                   |
 | [sys/replication/dr/primary/secondary-token](/vault/api-docs/system/replication/replication-dr#generate-dr-secondary-token)                            | POST              | Generate a DR secondary activation token                                                                                                            |
-| [sys/rotate](/vault/api-docs/system/rotate)                                                                                                            | PUT               | Trigger a rotation of the backend encryption key                                                                                                    |
-| [sys/seal](/vault/api-docs/system/seal)                                                                                                                | PUT               | Seals the Vault                                                                                                                                     |
-| [sys/step-down](/vault/api-docs/system/step-down)
+| [sys/rotate](/vault/api-docs/system/rotate)                                                                                                            | POST              | Trigger a rotation of the backend encryption key                                                                                                    |
+| [sys/seal](/vault/api-docs/system/seal)                                                                                                                | POST              | Seals the Vault                                                                                                                                     |
+| [sys/step-down](/vault/api-docs/system/step-down)                                                                                                      | POST              | Forces a node to give up active status                                                                                                              |
+| [sys/storage/raft/snapshot-auto/config](/vault/api-docs/system/storage/raftautosnapshots#list-automated-snapshots-configs)                             | LIST              | Lists named configurations                                                                                                                          |
+| [sys/storage/raft/snapshot-auto/config/:name](/vault/api-docs/system/storage/raftautosnapshots)                                                        | POST, GET, DELETE | Creates or updates a named configuration                                                                                                            |
 
```

Comment:
Since we're keeping the HTTP verb column, adding the following could be useful:

```suggestion
~> **Note:** Vault treats the HTTP POST and PUT verbs as equivalent, so for each mention of POST in the table above, PUT may also be used. Vault uses the non-standard LIST HTTP verb, but also allows list requests to be made using the GET verb along with `?list=true` as a query parameter, so for each mention of LIST in the table above, GET with `?list=true` may also be used.
```

Reply:
Added to upcoming commit.